pkg/login/printClusterInfo.go

```diff
@@ -0,0 +1,75 @@
+package login
+
+import (
+	"fmt"
+
+	"github.com/openshift/backplane-cli/pkg/ocm"
+	logger "github.com/sirupsen/logrus"
+)
+
+//displayClusterInfo retrieves and displays basic information about the target cluster.
+
+func PrintClusterInfo(clusterID string) error {
+	logger := logger.WithField("clusterID", clusterID)
+
+	// Retrieve cluster information
+	clusterInfo, err := ocm.DefaultOCMInterface.GetClusterInfoByID(clusterID)
+	if err != nil {
+		return fmt.Errorf("error retrieving cluster info: %w", err)
+	}
+
+	// Display cluster information
+	printClusterField("Cluster ID:", clusterInfo.ID())
+	printClusterField("Cluster Name:", clusterInfo.Name())
+	printClusterField("Cluster Status:", clusterInfo.State())
+	printClusterField("Cluster Region:", clusterInfo.Region().ID())
+	printClusterField("Cluster Provider:", clusterInfo.CloudProvider().ID())
+	printClusterField("Hypershift Enabled:", clusterInfo.Hypershift().Enabled())
+	printClusterField("Version:", clusterInfo.OpenshiftVersion())
+	GetLimitedSupportStatus(clusterID)
+	GetAccessProtectionStatus(clusterID)
+
+	logger.Info("Basic cluster information displayed.")
+	return nil
+}
+
+func GetAccessProtectionStatus(clusterID string) string {
+	ocmConnection, err := ocm.DefaultOCMInterface.SetupOCMConnection()
+	if err != nil {
+		logger.Error("Error setting up OCM connection: ", err)
+		return "Error setting up OCM connection: " + err.Error()
+	}
+	if ocmConnection != nil {
+		defer ocmConnection.Close()
+	}
+	enabled, err := ocm.DefaultOCMInterface.IsClusterAccessProtectionEnabled(ocmConnection, clusterID)
+	if err != nil {
+		fmt.Println("Error retrieving access protection status: ", err)
```

Comment:
Why we have both `fmt.Println` and `return` for the same error message here? Is that duplicated?

The caller function `PrintClusterInfo` already receives the error from return. It is capable of handling or printing the error itself.